app/grandchallenge/hanging_protocols/templates/hanging_protocols/hangingprotocol_list.html

```diff
@@ -49,6 +49,7 @@ <h5 class="card-title">
             <p>No hanging protocols available.</p>
         {% endfor %}
     </div>
+    {% include "grandchallenge/partials/pagination.html" %}

```

Comment:
Hah, is this all it required? That was easy! =)

Reply:
Indeed! :)